DOCS/man/options.rst

```diff
@@ -2786,13 +2786,17 @@ Subtitles
 
     Default: sdr
 
+    This option is disabled when ``--blend-subtitles=no`` and ``--target-colorspace-hint=no`` are both used.
+
 ``--sub-hdr-peak=<sdr|10-10000>``
     Controls the text subtitle and OSD diffuse white level in cd/m² (nits)
     for HDR output. ``sdr`` is 203 cd/m² for standard SDR white.
     (``--vo=gpu-next`` only)
 
     Default: sdr
 
+    This option is disabled when ``--blend-subtitles=no`` and ``--target-colorspace-hint=no`` are both used.
```

Comment:
```suggestion
    It also affects subtitle brightness on HDR tone mapping with ``blend-subtitles=yes/video``. 
```

Reply:
Isn't it just the altnative expression of `Controls the text subtitle and OSD diffuse white level in cd/m²`?

Reply:
The modified expression complements the `for HDR output`, indicating that the `blend-subtitles=yes/video` option in the HDR tone mapping will mix subtitles into the HDR color space.

Your original formulation was just a disguised way of stating that the option wouldn't take effect when subtitles were output in non-HDR, which is another way of expressing the obvious to `for HDR output`.

Reply:
I don't understand.

Reply:
No, what I'm saying is that you are wrong in this description: `This option is disabled when ``--blend-subtitles=no`` and ``--target-colorspace-hint=no`` are both used`.
Because of this: 
> On SDR monitors and HDR monitors in SDR mode, the `--target-colorspace-hint=auto` has the same behavior as `--target-colorspace-hint=no`.
> This option also doesn't take effect when the monitor is in SDR display mode when using `--target-colorspace-hint=auto --blend-subtitles=no`, because the subtitles are not in HDR output.

Reply:
要不你说中文吧，我真的get不到你的意思

Reply:
SDR 模式下等效 no 不代表你可以在描述里不提及。

> auto 就是可能会给你选择一个错误的输出

正如我上面有提到的，auto 只在 vulkan 下可能选择错误输出，这已在 https://github.com/mpv-player/mpv/pull/16512 中修复

Reply:
这与你的描述不准确无关好吧，请不要偏离主题

Reply:
我的观点已如前所述，auto的行为不属于这个PR

Reply:
`处于 SDR 显示模式下的 --target-colorspace-hint=auto --blend-subtitles=no`
如果合成器有问题导致的选择了hdr输出，那么此时sub-peak就有效

Reply:
`即处于 SDR 显示模式下的 --target-colorspace-hint=auto --blend-subtitles=no`
你对这个表述有更好的语言就提出。
如前所述，我也说了你的观点是补充。

Reply:
当`--target-colorspace-hint=auto  --blend-subtitles=no`选择SDR输出时此选项也无效